ml_dtypes/tests/custom_float_test.py

```diff
@@ -181,6 +198,7 @@ def dtype_has_inf(dtype):
             range(1 << n, 2 << n, 1 << max(0, n - 2)) for n in range(16)
         )
     ),
+    float8_e8m0fnu: [0, 2, 4],
```

Comment:
Zero value cannot be represented by E8M0 type, so it cannot round trip to int and back. I wonder why this _only_ failed on MacOS.

Reply:
Thanks, indeed my bad on this one. Weird it did not raise an error on Linux builds though.